terraform/modules/gro_ingestion_service/lambdas/src/functions/publishEvent.function.ts

```diff
@@ -58,36 +59,37 @@ const publishEvent = async (event: PublishEvent, accessToken: string) => {
     const options: RequestOptions = {
         method: "POST",
         headers: {
-            "Authorization": `Bearer ${accessToken}`,
+            Authorization: `Bearer ${accessToken}`,
             "Content-Type": "application/json; charset=utf-8",
-            "Content-Length": eventData.length
+            "Content-Length": eventData.length,
         },
     }
 
     return await makeRequest(gdxUrl, options, eventData)
-        .then(response => ({ success: true, result: response, event }))
-        .catch(error => ({ success: false, result: error, event }))
+        .then((response) => ({ success: true, result: response, event }))
```

Comment:
Is this a prettier thing? I don't see why we need the added `()` here or below

Reply:
I'll see if I can suppress

Reply:
done